contracts/0.4.24/utils/Versioned.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: 2023 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.4.24;
+
+import "@aragon/os/contracts/common/UnstructuredStorage.sol";
+
+
+contract Versioned {
+    using UnstructuredStorage for bytes32;
+
+    event ContractVersionSet(uint256 version);
+
+    /// @dev Storage slot: uint256 version
+    /// Version of the initialized contract storage.
+    /// The version stored in CONTRACT_VERSION_POSITION equals to:
+    /// - 0 right after the deployment, before an initializer is invoked (and only at that moment);
+    /// - N after calling initialize(), where N is the initially deployed contract version;
+    /// - N after upgrading contract by calling finalizeUpgrade_vN().
+    bytes32 internal constant CONTRACT_VERSION_POSITION = keccak256("lido.Versioned.contractVersion");
+
+    uint256 internal constant PERTIFIED_VERSION_MARK = uint256(-1);
```

Comment:
Let's refactor 0.9.8 version in accordance?
- add `PERTIFIED_VERSION_MARK`
- replace `getContractVersion() external` and `_getContractVersion()` by `getContractVersion() public`

- I'd also like to refactor this in 0.8.9 version:
```
        if (_getContractVersion() == 0) {
            _writeContractVersion(1);
        } else {
            revert NonZeroContractVersionOnInit();
        }
```
to the common "require-like" style:
```
        if (_getContractVersion() != 0) {
            revert NonZeroContractVersionOnInit();
        }
        _writeContractVersion(1);
```